openapi.yaml

```diff
@@ -28624,9 +30121,7 @@ components:
           description: 'One or more lists of values indexed by header name that will cause this Route to match if present in the request. The `Host` header cannot be used with this attribute: hosts should be specified using the `hosts` attribute. When `headers` contains only one value and that value starts with the special prefix `~*`, the value is interpreted as a regular expression.'
           type: object
           additionalProperties:
-            items:
```

Comment:
This looks like a breaking change for the `konnect_gateway_route` resource - running the e2e tests errored out 
Is this intended?
@mheap

Reply:
No, this is not an intended change. Nor is the `ordering` change above. Let me investigate

Reply:
@harshadixit12 Try the `admin-spec-updates` branch on `platform-api`. I've fixed the generator

Reply:
@mheap the [admin-spec-updates](https://github.com/Kong/platform-api/pull/1133/files) branch doesn't seem to fix this though - `konnect_gateway_route` resource still expects string for `headers.Accept` instead of array of strings (e2e tests break)

Reply:
Sorry, that's another fix I forgot about. Remove the following from the CPC overlay:

```
  - target: $["components"]["schemas"]["RouteJson"]["properties"]["headers"]["additionalProperties"]
    remove: true
  - target: $["components"]["schemas"]["RouteJson"]["properties"]["headers"]
    update:
      additionalProperties:
        type: string
```

Reply:
That fixed it, thanks 😄 
I'll send a PR to remove the overlay on platform-api